### ISSUE TITLE
Apps external to k8s

### ISSUE THREAD

**Issue by brackend** [OpenUnison/openunison-k8s-login-activedirectory#108]
Is it possible to provide authorization to applications outside of k8s as well as to internal?

For example, github or gitlab

**Comment by brackend** [comment]
Thank you Mark

**Comment by brackend** [comment]
Planning to try this with Gitlab in coming week or so.
But just tired it with Grafana and worked perfectly! Very happy with this.   I had it working with previous releases but with more recent release failed.  I just copied the links the files you shared and updated.   Can I be lame and repeat again at how so very cool this is!

The main difference was:  authChainName: **login-service**  whereas previously it was  somthing like: **loginservice**

**Comment by mlbiam** [comment]
ahh, i forgot about that.  i'll update our upgrade docs.  We had to rename many of the configurations because k8s requires names to be lower case.  For GitLab, here's an example we used at the end of Kubernetes: An Enterprise Guide 2nd Ed - https://github.com/PacktPublishing/Kubernetes---An-Enterprise-Guide-2E/tree/main/chapter14/naas-gitops/templates/gitlab-sso

**Comment by brackend** [comment]
For some external services you may not want to share all groups.   Is there a way to filter which groups reach the external web application.

**Comment by mlbiam** [comment]
Yes.  You would need to create a new idp configuration.  I'll write up a tutorial.  I'm assuming oidc?  Also, do your groups have a naming standard you'd like to limit by?

**Comment by brackend** [comment]
Thank you.  That would be great.  With new groups we could create a naming standard to differentiate them from existing groups.

Yes oidc

**Comment by mlbiam** [comment]
Haven't forgotten about this. Realized I needed to add a feature to make this work the way I wanted in k8s. Cutting the new release tomorrow and publishing the doc/video too

**Comment by mlbiam** [comment]
update all your charts and make sure you're running the latest containers - https://openunison.github.io/documentation/custom-sso/
